tests/nlu/base/test_training_data.py

```diff
@@ -505,36 +506,39 @@ def test_markdown_entity_regex():
     """
 
     result = r.reads(md)
+    expected_texts = [
```

Comment:
Why was `test_markdown_entity_regex` modified? Is it related to the changes in `training_data.py`?

Reply:
Initially, the fix to `training_data.py` used a set to remove duplicates so order is lost, which is why I rewrote the `test_markdown_entity_regex`. I don't think this is needed anymore since the current solution maintains order.

Can I just revert commit `0330f26` or will this cause problems? I am not very familiar with git.

Reply:
Hey @hsm207, yes, you can! It's actually pretty easy:
```bash
$ git revert 0330f26
```
This will create a new commit which will have an exact opposite diff of `0330f26`. It's equivalent to undoing the changes yourself and then commiting the results. The new commit will have a message similar to "Revert commit <commit text for 0330f26>".

Reply:
thanks for the tip @federicotdn 
I've reverted the commit